scrapy/czlscrape/utils.py

```diff
@@ -0,0 +1,15 @@
+import re
+
+
+def guess_publication_type(text, rules):
+    text = text.lower()
+    text = re.sub(r'[șş]', 's', text)
+    text = re.sub(r'[țţ]', 't', text)
+    text = re.sub(r'[ăâ]', 'a', text)
+    text = re.sub(r'[î]', 'i', text)
+
+    for substr, publication_type in rules:
```

Comment:
Da, spre exemplu intre dialog si comunicatii difera. Poate ca sunt si alte situatii, n-am verificat. Legat de guessing la receiver - in comunicatii am nevoie de type ca identific titlurile si sa separ articolele, nefiind absolut nicio structura. Daca scoatem logica asta din spider, nu mai pot separa articolele.